enterprise/backend/src/metabase_enterprise/workspaces/isolation.clj

```diff
@@ -21,15 +22,15 @@
 (defn- isolation-schema-name
   "Generate schema/database name for workspace isolation following mb__isolation_<slug>_<workspace-id> pattern."
   [workspace-id]
-  (let [instance-slug (instance-uuid-slug (str (system/site-uuid)))
+  (let [instance-slug      (instance-uuid-slug (str (system/site-uuid)))
```

Comment:
You'll see I align things in all the files. I find it easier to read, and my editor does it automatically.

Perhaps there's a way to configure this in your editors too.